### PR TITLE
fix(notifications/manual-judgements): don't let echo work wrap slack messages to 80 chars

### PR DESCRIPTION
**Pull request by kevinawoo**

### CODE REVIEW

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package com.netflix.spinnaker.echo.notification; /*
```
Comment (link108) [inline echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java]: package after copyright

Comment (jeffhart-armory) [inline echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java]: Your copyright and import statements are a little messed up.

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        append("\n");
+      }
+    }
+
+    // hit when all of the node's children (if any) have been visited
+    public void tail(Node node, int depth) {
+      String name = node.nodeName();
+      if (StringUtil.in(name, "br", "dd", "dt", "p", "h1", "h2", "h3", "h4", "h5")) {
+        append("\n");
+      } else if (name.equals("a")) {
```
Comment (link108) [inline echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:59]: should this if/else be reversed? I don't see a default case, so am wondering if it'd be better to do:

```
if (name.equals("a")) {
  append(String.format(" <%s>", node.absUrl("href")));
} else {
   append("\n");
}
```
but you may want to skip everything that doesn't match that list, so depends on what you want to do

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java (hunk 1 above)
Comment (kevinawoo) [reply echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java]: damn you gradlew... what are you doing!?

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java (hunk 2 above)
Comment (kevinawoo) [reply echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:59]: 🤷‍♂ 	copy and paste job from https://github.com/jhy/jsoup/blob/jsoup-1.8.2/src/main/java/org/jsoup/examples/HtmlToPlainText.java

Comment (kevinawoo) [reply echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:59]: i guesssssssssss I should fix it >_<

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        append(String.format(" <%s>", node.absUrl("href")));
+      }
+    }
+
+    // appends text to the string builder
+    private void append(String text) {
+      if (text.equals(" ")
```
Comment (link108) [inline echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:66]: may want to clean this up so its easier to read. Can do so in two ways: 

1. Make a function for the test 
2. Add parentheses so the order of operations is easier to read without thinking too hard about :)

eg:
```
if ( text_and_accum_empty || just_newlines) {
  return;
}
```

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java (hunk 2 above)
Comment (kevinawoo) [reply echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:59]: oh actually, we just drop the tail, (kind of unexplicitfully)

File: echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java (hunk 3 above)
Comment (link108) [reply echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java:66]: ah I see the left paren on the or clause, spotless messed this up!

**Review by link108 — APPROVED** [review]

**Comment by kevinawoo** [comment]
https://github.com/spinnaker/echo/pull/616 is the real pr

**Pull request closed without merging**